lib/core/presentation/pages/event/guest_event_detail_page/guest_event_detail_page.dart

```diff
@@ -63,6 +64,11 @@ class _GuestEventDetailView extends StatelessWidget {
 
           final isAttending =
               EventUtils.isAttending(event: event, userId: userId);
+          final isOwnEvent =
+              EventUtils.isOwnEvent(event: event, userId: userId);
+          if (isOwnEvent) {
+            return const PostHostEventDetailView();
```

Comment:
`Post` as a prefix is not necessary in this context

Reply:
should create another event page for host. Should not combine all views in one page.

Reply:
Thanks, this super helpful